features/attached_commands.feature

```diff
@@ -515,8 +515,11 @@ Feature: Command behaviour when attached to an UA subscription
               {
                 "code": 200,
                 "response": {
```

Comment:
I interpreted the conversation as the new response would only be:
```json
{
  "response": {
    "activityToken": "test",
    "activityID": "ac-id",
    "activityPingInterval": "ping-test"
  }
}
```

Might be worth double checking.

And I'd expect this to maybe cause more changes necessary in contract.py `report_machine_activity`?

Reply:
That was the original idea, but I think me and @jaycee agreed that we should create the `activityInfo` structure on all endpoints. But it is worth to double check that.

@jaycee Is my interpretation correct ?